root/etc/e-smith/templates/etc/postfix/main.cf/30smtpauth_tls20180330

```diff
@@ -1,8 +1,8 @@
 {
  my $tlsPolicy = $tls{'policy'} || '0';
 
- if ($tlsPolicy >= '20180330') {
- $OUT .= '
+ if ($tlsPolicy == '20180330') {
```

Comment:
`==` operator compares two integers. Should be like [here](https://github.com/NethServer/nethserver-httpd/pull/28/files#diff-2cd0260994986ba81c6129e469c01bc9R4).